app/javascript/react/components/Calendar/Calendar.style.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+
+import { grey100, grey400 } from "../../assets/styles/colors";
+
+interface dayProps {
+  color?: string;
```

Comment:
A transient props as well, only relevant for styling purposes.

Reply:
I skipped color because I read that transient props are usually used for Boolean props. 
Changing it, but could you elaborate on "only relevant for styling purposes"?

Reply:
I see, from SC docs:
```
If you want to prevent props meant to be consumed by styled components from being passed to the underlying React node or rendered to the DOM element, you can prefix the prop name with a dollar sign ($), turning it into a transient prop.
```
https://styled-components.com/docs/api#transient-props